src/components/ui/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,60 @@
+import ReactPaginate from "react-paginate";
+import clsx from "clsx";
+
+import Button from "../Button";
+import Icon from "../Icon";
+
+import style from './Pagination.module.css'
+
+interface PaginationProps {
+    currentPage:number ;
+    totalPage:number;
+    onPageChange: (value: number)=>void; 
+    containerClassName?:string
+}
+
+const Pagination = ({currentPage, totalPage, onPageChange, containerClassName}:PaginationProps) => {
+
+    const handlePageChange = ({ selected }: { selected: number })=>{
+        onPageChange(selected + 1)
+    }
+
+    return <ReactPaginate
+        containerClassName={clsx(style.pagination, containerClassName)}
+        activeClassName={style.activePage}
+        previousClassName={style.previous}
+        nextClassName={style.next}
+        pageRangeDisplayed={2}
+        previousLabel={
+          <Button
+            size="fit"
+            variant="icon"
+            disabled={currentPage === 1}
+            className={style.paginationButton}
+          >
+            <Icon id="icon-dropdown" fill="var(--primary-dark)" size={28} />
+          </Button>
+        }
+        nextLabel={
+          <Button
+            size="fit"
+            variant="icon"
+            disabled={currentPage === totalPage}
+            className={style.paginationButton}
+          >
```

Comment:
i think you'd better to add icon in button with the help of icon prop

Reply:
Ok)